### PR TITLE
correct licensing

### PR DESCRIPTION
**Pull request by zo-el**

### CODE REVIEW

**Comment by zo-el** [comment]
@mkozjak  can I get a review?

File: README.md [hunk 1]
```diff
@@ -21,7 +22,7 @@
 </div>
 
 ## About
-
+This is a fork of [rpc-websocket](https://github.com/elpheria/rpc-websockets)
```
Comment (mkozjak) [inline README.md]: rpc-websockets*

**Review by mkozjak — APPROVED** [review]
Please see the comments. Looks good.
Please rewrite the history so we're not having any previous commits under a wrong license violating my work. You can squash everything into a single commit after this is merged in. All commits must use my license. Thanks!

**Comment by mkozjak** [comment]
Also remove existing tags, releases on git and unpublish npm/yarn packages if you have any. After this is rewritten, you can continue publishing releases as usual!

**Pull request merged**